python/ctranslate2/converters/transformers.py

```diff
@@ -2039,4 +2039,16 @@ def main():
         (26, 12),
         (27, 15),
     ],
+    "openai/whisper-large-v3": [
```

Comment:
Possible worth adding a comment on the source of these since its different from the source on L1928.